cmd/log-parser/README.md

```diff
@@ -37,7 +37,7 @@ The tool requires that the following fields are defined for each log record:
 - Source field (`source`): a single word that specifies the name of a unique
   part of the system (e.g. `proxy`, `runtime`, `shim`).
 
-- Timestamp field (`time`): in [RFC3339](https://www.ietf.org/rfc/rfc3339.txt)
+- Timestamp field (`time`): in [RFC3339]
```

Comment:
I may have found an alternative link
https://tools.ietf.org/html/rfc3339
Or we might have to go for ISO 8601 - but, I'd have to ask @jodh-intel how that fits in, or not :-)
https://en.wikipedia.org/wiki/ISO_8601